proofs/tree.go

```diff
@@ -280,6 +280,19 @@ func NewDocumentTree(proofOpts TreeOptions) DocumentTree {
 	}
 }
 
+// NewDocumentTree returns a DocumentTree with `rootHash`
```

Comment:
```suggestion
// NewDocumentTree returns a DocumentTree with that has a root hash set. It can be used to validate proofs but not for creating any. 
```